pkg/api/pod/util.go

```diff
@@ -19,7 +19,7 @@ package pod
 import (
 	"strings"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit/curious for myself: why add the explicit v1 here?

Reply:
this is done by VScode automatically =) I see it in many PRs. Happy to remove if this is annoying.

Reply:
Nope, no need - was just curious!